content/docs/engine-dev/assets/filetypes/texture-support.md

```diff
@@ -0,0 +1,113 @@
+#### Why is this concern?
+
+Cache consistency is very important. Situations like this can break user trust in asset processing, create situations where projects work differently on different team members machines, and the time between causing the issue (renaming the texture) and discovering the bug (the reference is broken, the scene builder can't find the texture and the azmaterial product asset no longer has a valid texture) may take a long time. If everyone on the team is working with iterative caches day to day, and someone finally clears their cache or sets up a new project several weeks after the rename, that's a lot of other changes that could have caused the problem.
+#### What can we do to address this?
+
+* Update scene files to emit job dependencies on referenced source asset image files.
+* Make sure job dependencies are using asset IDs instead of paths before being recorded into the asset database, so that renaming an image file will cause the dependency job for the new name to match the old dependency, and cause the scene files to reprocess.
+
+This will mean that renaming an image referenced by an FBX file will cause that FBX file to reprocess, due to the job dependency. This will maintain cache consistency for a team, and make it easy to quickly find issues when an image was referenced by an FBX file, so the team can go upstream to adjust the Blender, Maya, or other working asset to point at the new image, and re-export the FBX with the new path.
+### Scene file source assets have path based references to images
+
+This is a known issue with scene files referencing images, users tend to expect this.
+### UI references to images via simple asset reference
+
+UI components tend to use SimpleAssetReference to reference files, instead of actual asset references. This is because the UI system and components were built before proper asset referencing was available.
+
+This includes the texture atlas system, because it's primarily used by UI.
+
+#### Why is this not a major problem?
+
+The UI system predates a lot of current systems and ideally it will be overhauled to use modern solutions (asset references instead of simple asset references, prefabs instead of slices, etc), which would automatically handle this problem.
+#### Why is this still a concern?

```

Comment:
```suggestion
The UI system predates a lot of current systems and ideally it will be overhauled to use modern solutions (asset references instead of simple asset references, prefabs instead of slices, etc), which would automatically handle this problem.

#### Why is this still a concern?

```